jakovleva_ju/task2/Source.cpp

```diff
@@ -52,17 +52,17 @@ DynamicArr::DynamicArr(DynamicArr& x)
 }
 DynamicArr::~DynamicArr()
 {
-	if (arr != NULL) free(arr);
+	delete(arr);
```

Comment:
Неверно. Должно быть так:
```suggestion
	delete [] arr;
```